spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/SpanCollectionHealthIndicator.java

```diff
@@ -0,0 +1,26 @@
+package org.springframework.cloud.sleuth.health;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+
+/**
+ * Health indicator that shows the number of accepted / dropped spans.
+ *
+ * //TODO: should this be a health indicator (it's never down) or an Endpoint ?
+ *
+ * @author Marcin Grzejszczak
+ */
+public class SpanCollectionHealthIndicator extends AbstractHealthIndicator {
```

Comment:
I'd really skip this for now, and just focus on collecting metrics. Let's leave health, thresholds etc. in the space of metrics systems (as opposed to a hard-wired aspect of `SpanCollectorMetricsHandler`). If someone wants to read stats like below, they could just as well look at their metrics endpoint locally or otherwise.